frontend/taipy-gui/src/context/taipyReducers.ts

```diff
@@ -108,6 +109,10 @@ interface TaipyMultipleMessageAction extends TaipyBaseAction {
 
 interface TaipyAlertAction extends TaipyBaseAction, AlertMessage {}
 
+interface TaipyDeleteAlertAction extends TaipyBaseAction {
+    notification_id: string;
```

Comment:
I would even say:
```suggestion
    notification_id?: string;
```
Unidentified notifications cannot be revoked specifically.

Reply:
if we delete we have to know what we're deleting